app/policies/partner_policy.rb

```diff
@@ -31,9 +31,8 @@ def edit?
 
   def destroy?
     return true if user.root?
-    return false unless user.neighbourhood_admin?
-
-    user.owned_neighbourhood_ids.include?(record.neighbourhood_id)
+    return true if user.admin_for_partner?(record.id)
+    return true if user.only_neighbourhood_admin_for_partner?(record.id)
```

Comment:
(you could leave off the `return true if` part)

Reply:
have had weird issues with another policy where not returning true explicitly causes it to break so I'm going to leave this in? Also it kind of helps my brain to just see the list of return statements!

Reply:
oh i do like a good bit of symmetry as well, hehehe